### PR TITLE
Clearify readme

### PR DESCRIPTION
**Pull request by kylecrum**
Getting a decent number of questions on this portion of the interview, so here's an attempt at clearifying some of the language.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,20 +3,22 @@
 This is a code extension designed so that we can code together during an on-site interview. Our guiding principal is: **we want to see you at your best** .
 
 ## Instructions
-If you bring your own computer to the interview, please ensure you are ready to go by having anything installed beforehand. If you want to use one of Reverb's computers, please tell us which code editor you prefer and which language you will be doing the extension in.
 
-## Details
+1. Choose which example (ruby or React) you would like to extend. If you don't feel comfortable in either of those 2 examples, feel free to implement your own (see "Implementing Your Own Solution"). Again, _we want to see the best side of you_ so please make sure you are comfortable with what we will be working on.
```
Comment (ebenoist) [inline README.md]: I'll make sure to get my go one up and going by next week as well

File: README.md [hunk 2]
```diff
@@ -3,20 +3,22 @@
...
+4. That's it! We'll give you all other information during the coding session.
 
-1. The extension solves "The Problem" stated below.
+## Words of Wisdom
 
-2. We have provided some starting points in some languages, but if you would rather use a different language or use something else to solve the problem below, please do. Again, _we want to see the best side of you_ so please make sure you are comfortable with what we will be working on.
+1. Our implementations are not 100% complete or refactored in the best possible way. None of the code is gospel, so don't think you have to copy our style in any way. We're open to new ways of doing things as long as we can have a dialogue about it.
 
-3. Our implementations are not 100% complete or refactored in the best possible way. None of the code is gospel, so don't think you have to copy our style in any way.
+2. Our intention is not to trick you or test you; we want to collaborate. So if you have any questions about this, please reach out and ask!
```
Comment (ebenoist) [inline README.md]: 👍

File: README.md [hunk 3]
```diff
@@ -3,20 +3,22 @@
...
 
-4. Our intention is not to trick you or test you; we want to collaborate. So if you have any questions about this, please reach out and ask!
+### Implementing Your Own Solution
+Again, _you do not need to do this_, but if ruby / React aren't your jams, we don't want to force you into something you don't know. So, if you'd like to use another language / framework, we welcome that. In that case, please come to the office with code that solves "Problem" stated below.
```
Comment (ebenoist) [inline README.md]: *nit* Ruby should probably be capitalized?

Comment (ebenoist) [reply README.md]: "please come to the office with code that solves **the** "Problem" stated below."

Comment (kylecrum) [reply README.md]: prolly

File: README.md (hunk 1 above)
Comment (kylecrum) [reply README.md]: ya, no worries. No one has specifically requested Go yet.

File: README.md (hunk 3 above)
Comment (kylecrum) [reply README.md]: Fixed both

**Pull request merged**